internal/extproc/server.go

```diff
@@ -168,6 +170,25 @@ func (s *Server) Process(stream extprocv3.ExternalProcessor_ProcessServer) error
 
 		resp, err := s.processMsg(ctx, p, req)
 		if err != nil {
+			s.logger.Info("AAAAA")
```

Comment:
The fact that routes are calculated is an implementation detail of the chat completion processor, because it's based on the request body for requests to that endpoint. Therefore, I would not have this logic here as it couples the `server` to semantics of a particular processor.

Let's keep this  unchanged and just have the chat completion processor return the immediate response.